api/controller/controller_test.go

```diff
@@ -350,3 +350,29 @@ func (s *Suite) TestModelStatusError(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "model error")
 	c.Assert(out, gc.IsNil)
 }
+
+func (s *Suite) TestConfigSet(c *gc.C) {
+	apiCaller := apitesting.BestVersionCaller{
+		BestVersion: 5,
+		APICallerFunc: func(objType string, version int, id, request string, args, result interface{}) error {
```

Comment:
we generally check the objType and request args as well

Reply:
Will do.